client-v2/src/shared/images/icons-hover/index.js

```diff
@@ -0,0 +1,17 @@
+import View from './icon-hover-view.svg';
+import Ophan from './icon-hover-ophan.svg';
+import Copy from './icon-hover-copy.svg';
+import Paste from './icon-hover-paste-sublink.svg';
+import Clipboard from './icon-hover-clipboard.svg';
+import Delete from './icon-hover-delete.svg';
+
+const hoverActionIcons = {
```

Comment:
We could export these as named exports like `export { default as view } from './icon-hover-view.svg';` which just means that webpack will tree shake any unused SVGs in future. When they're exported on an object then webpack can't do this. It'll probably be fine but perhaps worth guarding against.

Reply:
I have made this change.